1_concepts/1_1_default_clone_copy/README.md

```diff
@@ -71,7 +71,9 @@ __Estimated time__: 1 day
 
 
 - Create a `Point` type which represents a 2D point (`x` and `y` coordinates). This type has to be `Copy` and `Default`.
-- Create a `Polyline` type which represents a non-empty set of `Point`s of unknown size. This type has to be `Clone` and non-`Default`.
+- Create a `Polyline` type which represents a non-empty collection(whichever you want) of `Point`s of unknown size. This type has to be `Clone` and non-`Default`.
+    - Polyline must implement basic collection methods: addition of an item, removal of an item,
```

Comment:
I mean it's not the point of the task. So I think it really should be optional.

Reply:
In my opinion, it would be enough to replace "set" with "collection" 🤔

Reply:
ok